packages/rax/src/vdom/instance.js

```diff
@@ -58,6 +59,20 @@ export default {
       }
     }
 
+    if (isWeb) {
+      // handle rendered ELement
+      if (container.childNodes) {
+        const childNodes = [...container.childNodes];
+
+        for (let i = 0; i < childNodes.length; i ++) {
+          const rootChildNode = childNodes[i];
+          if (rootChildNode.hasAttribute('data-rendered')) {
```

Comment:
text node do not has method hasAttribute, this should be `rootChildNode.hasAttribute && rootChildNode.hasAttribute('data-rendered')`